tripy/docs/conf.py

```diff
@@ -150,11 +151,19 @@ def process_docstring(app, what, name, obj, options, lines):
                     pname = "*" + pname
 
                 if pname == "self":
-                    # Don't want a type annotation for the self parameter.
-                    assert (
-                        param.annotation == signature.empty
-                    ), f"Avoid using type annotations for the `self` parameter since this will corrupt the rendered documentation!"
+                    if obj.__qualname__ in FUNC_W_DOC_VERIF:
```

Comment:
Can we not just use the `TYPE_VERIFICATION` map to check this?

Reply:
‘TYPE_VERIFICATION’ uses the name from ‘function_name’ parameter from ‘dtype_info’, so functions like ‘__radd__’ would get messed up if I don’t use a separate variable.